<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChunkDistributorVisualizer</class>
 <widget class="QMainWindow" name="ChunkDistributorVisualizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chunk Distributor Visualizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="Algo::Algorithm" name="t_algo">
       <property name="showDescription" stdset="0">
        <bool>true</bool>
       </property>
       <property name="priorityCount" stdset="0">
        <number>0</number>
       </property>
       <property name="AnimationDuration" stdset="0">
        <number>250</number>
       </property>
       <property name="showAnimation" stdset="0">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Page Filler Algorithmus</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="t_musi">
       <attribute name="title">
        <string>ATMDS3 Music Packer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0,0,0,10" columnstretch="0,1,0,0">
        <item row="4" column="3">
         <widget class="QLCDNumber" name="lcd_spd">
          <property name="smallDecimalPoint">
           <bool>false</bool>
          </property>
          <property name="digitCount">
           <number>3</number>
          </property>
          <property name="mode">
           <enum>QLCDNumber::Mode::Dec</enum>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::SegmentStyle::Filled</enum>
          </property>
          <property name="value" stdset="0">
           <double>999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Tune:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="le_lPrio">
          <property name="text">
           <string>85,86</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cb_tune"/>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QTextEdit" name="te_sizes">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;142,142,37,37,29,29,21,21,21,21,21,21,21,21,21,21,21,19,19,23,16,19,23,35,23,13,19,13,40,35,137,92,18,58,44,20,29,106,19,29,33,58,106,20,47,6,49,34,34,4,35,41,19,106,81,52,16,35,50,10,13,26,10,34,36,38,45,42,45,140,140,140,140,13,10,12,13,60,60,73,68,49,11,30,30,16,2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Low Prio:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="media-playback-start"/>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="sb_start">
          <property name="suffix">
           <string/>
          </property>
          <property name="prefix">
           <string>$</string>
          </property>
          <property name="minimum">
           <number>2049</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>6244</number>
          </property>
          <property name="displayIntegerBase">
           <number>16</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Startadresse:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Normal Prio:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>High Prio:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="le_hPrio">
          <property name="text">
           <string>0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,77,78</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="le_nPrio">
          <property name="text">
           <string>17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,79,80,81,82,83,84</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Größen:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="4">
         <widget class="QGraphicsView" name="v">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>5</number>
          </property>
          <property name="midLineWidth">
           <number>3</number>
          </property>
          <property name="interactive">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
          </property>
          <property name="dragMode">
           <enum>QGraphicsView::DragMode::ScrollHandDrag</enum>
          </property>
          <property name="transformationAnchor">
           <enum>QGraphicsView::ViewportAnchor::AnchorUnderMouse</enum>
          </property>
          <property name="resizeAnchor">
           <enum>QGraphicsView::ViewportAnchor::AnchorUnderMouse</enum>
          </property>
          <property name="viewportUpdateMode">
           <enum>QGraphicsView::ViewportUpdateMode::SmartViewportUpdate</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="4">
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,1">
          <item>
           <widget class="QRadioButton" name="rb_boosted">
            <property name="text">
             <string>silent</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bg</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_visual">
            <property name="text">
             <string>visual</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bg</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_animated">
            <property name="text">
             <string>animated</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bg</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sl_spd">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>999</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TickPosition::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>250</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="tb">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="sb"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Algo::Algorithm</class>
   <extends>QWidget</extends>
   <header>Algo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sl_spd</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_spd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1235</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>1225</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="bg"/>
 </buttongroups>
</ui>
